pandas/core/base.py

```diff
@@ -314,7 +314,13 @@ def _try_aggregate_string_function(self, arg, *args, **kwargs):
 
         f = getattr(np, arg, None)
         if f is not None:
-            return f(self, *args, **kwargs)
+            try:
+                return f(self, *args, **kwargs)
+            except AttributeError:
+                raise ValueError(
+                    "'{arg}' is not a valid function for "
+                    "'{cls}' object".format(arg=arg, cls=type(self).__name__)
+                )
 
         raise ValueError("{arg} is an unknown string function".format(arg=arg))
```

Comment:
Does this get hit any more?

Reply:
It does for single string argument or a dictionary. Other one is for list or tuple with invalid functions